V1.4 PE-D Bug fixes and UG updates

ZhiWei1010 opened the pull request:
This commit updates the User Guide as well as the Help Window UI.
* Update sortResult screenshot in UG to correctly display sort patients by preferred name from fresh data sample
* Update all command section to specify that screenshot displayed is for executing `sort p` command
* Update example in delete command to match with other example
* Update example in delete event command to work correctly with fresh sample data
* Update helpMessage screenshot in UG to correcty display the updated link in help message
* Update USERGUIDE_URL in UI#helpWindow to correctly display the link to user guide

Fixes #240 Fixes #231 Fixes #225

ZhiWei1010 committed:
Docs: Update typo error in help command of User Guide

ZhiWei1010 committed:
Merge branch 'master' into v1.4-PE-D-bug-fix-help-command

ZhiWei1010 committed:
Docs: Replace sortResult image to new sortpResult image

Update the sort result image to be sort result by preferred name from
fresh sample data and be consistent with other screenshot size.

ZhiWei1010 committed:
Docs: Update sort result section in User Guide

Add a new line to specify command used is sort p, corresponding to
example screenshot.

ZhiWei1010 committed:
UI: Update USERGUIDE_URL to correctly point to user guide

ZhiWei1010 committed:
Revert "UI: Update USERGUIDE_URL to correctly point to user guide"

This reverts commit 2b2581f31127c8051562c32416a0ff0a75db4614.

ZhiWei1010 committed:
Ui: Update USERGUIDE_URL to point to User Guide correctly

ZhiWei1010 committed:
Docs: Update helpMessage Screenshot to show updated link in UG

choowengyan commented on docs/UserGuide.md:
```diff
@@ -418,7 +418,9 @@ Examples:
 * `sort p`
 
-![result for 'sort'](images/sortResult.png)
+This screenshot shows the result of executing `sort p`.
```
To standardise this, may need to check if other command section contains the explanation of screenshot as well.

ZhiWei1010 replied:
After offline discussion, we decide to standardise and add this line to all other commands!

choowengyan commented on docs/UserGuide.md:
```diff
@@ -81,7 +81,7 @@ command to run the application.<br>
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
```
Good catch on the spelling error!

ZhiWei1010 replied:
It was raised in the PE-D #231 !

choowengyan reviewed (changes requested):
Overall, great changes made! Just to check if the explanation is needed for the rest of the doc mentioned.

ZhiWei1010 committed:
Docs: User Guide standardization

Add a line to specify the command executed for each screenshot.

Update example in delete command to be consistent with other example
and match with screenshot.

Update example in deletee command to ensure that it works with
fresh sample data.

ZhiWei1010 committed:
Docs: Replace deleteEventResult screenshot with updated copy

ZhiWei1010 committed:
Docs: Replace listResult screenshot with updated copy

choowengyan reviewed (approved):
LGTM! Well done on the screenshot description!
